packages/Breadcrumbs/src/components/Link/Link.js

```diff
@@ -85,6 +92,7 @@ const defaultProps = {
 };
 
 Link.displayName = "Breadcrumbs.Link";
+Link.Tooltip = LinkTooltipContentPropsCollector;
```

Comment:
Probably this should go _after_ the `defaultProps` for consistency, but 🤷‍♂️  Not worth rerunning Semaphore for.